lib/muffin/base.rb

```diff
@@ -19,5 +19,29 @@ def initialize(request: nil, params: nil, scope: nil)
 
       assign_attributes
     end
+
+    private
+
+    def update_nested!(relation, entities)
+      entities = [entities] unless entities.is_a?(Array)
+
+      entities.each do |entity|
+        entity_attributes = entity.attributes.except(:_destroy)
+
+        if entity.respond_to?(:id) && entity.id.present?
+          model = relation.find(entity.id)
+
+          if entity.respond_to?(:_destroy) && entity._destroy
+            model.destroy
+          else
+            model.update_attributes!(entity_attributes)
```

Comment:
you should also check if the attributes are actually updated. if attributes are not changed, also the `updated_at` should not change.